dnf/db/history.py

```diff
@@ -56,18 +56,27 @@ def name(self):
         return self._item.getRPMItem().getName()
 
     @property
-    def evr(self):
-        epoch = self._item.getRPMItem().getEpoch()
-        version = self._item.getRPMItem().getVersion()
-        release = self._item.getRPMItem().getRelease()
-        if epoch:
-            return "{}-{}".format(version, release)
-        return "{}:{}-{}".format(epoch, version, release)
+    def epoch(self):
+        return self._item.getRPMItem().getEpoch()
+
+    @property
+    def version(self):
+        return self._item.getRPMItem().getVersion()
+
+    @property
+    def release(self):
+        return self._item.getRPMItem().getRelease()
 
     @property
     def arch(self):
         return self._item.getRPMItem().getArch()
 
+    @property
+    def evr(self):
+        if self.epoch:
+            return "{}-{}".format(self.version, self.release)
+        return "{}:{}-{}".format(self.epoch, self.version, self.release)
```

Comment:
shouldn't this condition be other way round?

Reply:
correct :)
let me fix it